src/pydata_sphinx_theme/assets/styles/components/_switcher-version.scss

```diff
@@ -72,6 +72,10 @@ button.version-switcher__button,
   font-size: 1.1em; // A bit smaller than other menu font
   z-index: $zindex-modal; // higher than the sidebars
 
+  // Make sure it meets WCAG target size requirement no matter the version
+  // string displayed in the button
+  @include min-hit-area;
```

Comment:
This pull request doesn't do a lot of work on the version switcher because it's not an icon link. However, since a version string could be very small maybe even empty and in that case the button might fall below the minimum width, so I thought it would be a good idea to add this precaution.